### DISCUSSION TITLE
COMO ELIMINAR LA CREACION DE UN ARCHIVO .EXE AL EJECUTAR MI CODIGO EN C++

### DISCUSSION THREAD

**Discussion by EdrianAdruZ** [microsoft/vscode-cpptools#12857] [General]
Al crear un archivo con la extencion .c++ y ejecutarlo este se ejecuta y me crea un archivo .exe con el mismo nombre que el archivo.
Por ejemplo:
Si creo un archivo con el nombre : "pruebaUno.c++" me crea otro archivo "pruebaUno.exe"
Me gustaria suprimir la creacion de ese archivo .exe alguien podria ayudarme solo me pasa en archivos .c++

**Comment by sean-mcmanus** [comment]
@EdrianAdruZ  You can't suppress the creation of the .exe file -- that is the binary output result that is run. You could potentially modify it's name though in your tasks.json, i.e.

```json
        "-o",
        "${fileDirname}\\${fileBasenameNoExtension}.exe"
```
